src/peft/utils/other.py

```diff
@@ -313,6 +316,28 @@ def _get_batch_size(input_ids: Optional[torch.Tensor], inputs_embeds: Optional[t
     return batch_size
 
 
+def default_forward(self, *args, **kwargs):
+    return self.get_base_model()(*args, **kwargs)
+
+
+def update_forward_signature(model) -> Union[torch.nn.Module, PushToHubMixin]:
+    """
+    Updates the forward signature of the PeftModel to include parents class signature
+    """
```

Comment:
Having an example here would be super helpful to users.

Reply:
Added example with WhisperForConditionalGeneration